vaadin-date-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datepicker/DatePickerIT.java

```diff
@@ -147,45 +148,187 @@ public void selectDatesOnLinkedDatePickers() {
 
     @Test
     public void selectDatesOnCustomLocaleDatePickers() {
```

Comment:
This method was just converted from old style to use DatePickerElement and TestBenchElement. Nothing else was changed in this method. Then I used this as a basis for the new methods below.

Reply:
Would it make sense to group patterns like these into methods and reuse them instead of copy-pasting?
```java
        localePicker.setDate(LocalDate.of(900, Month.MARCH, 4));
        Assert.assertEquals("0900/3/4", localePicker.getInputValue());
```

```java
        localePicker.setInputValue("3/7/0900");
        Assert.assertEquals("3/7/0900", localePicker.getInputValue());
        Assert.assertEquals(LocalDate.of(900, Month.MARCH, 7),
                localePicker.getDate());
```

Reply:
I briefly thought about it but though that would just make the tests slightly harder to read/understand and not that much shorter.

I guess with some helpers the above could be something like:
```java
setDateAndAssert(localePicker, LocalDate.of(900, Month.MARCH, 4), "0900/3/4");
```

```java
setInputValueAndAssert(localePicker, "3/7/0900", LocalDate.of(900, Month.MARCH, 7));
```
or another example with one more param (when input value should be transformed):
```java
setInputValueAndAssert(localePicker, "3/7/20", "3/7/2020", LocalDate.of(2020, Month.MARCH, 7));
```

I guess I will try this approach and see what the tests will look like then.

Reply:
@tulioag I updated the new tests to use this kind of approach, PTAL.